webpack.config.js

```diff
@@ -1,23 +1,20 @@
 /* "Copyright © 2020, Alexandra Rose Charity (reg. in England and Wales, #00279157)" */
-var path = require('path');
-var webpack = require('webpack');
-var GitRevisionPlugin = require('git-revision-webpack-plugin');
-var HtmlWebpackPlugin = require('html-webpack-plugin');
-var CopyWebpackPlugin = require('copy-webpack-plugin');
-var OfflinePlugin = require('offline-plugin');
-var UglifyJsPlugin = require('uglifyjs-webpack-plugin');
-var { VueLoaderPlugin } = require('vue-loader');
-var WebpackPwaManifest = require('webpack-pwa-manifest');
-var gitRevisionPlugin = new GitRevisionPlugin();
+const path = require('path');
+const webpack = require('webpack');
+const GitRevisionPlugin = require('git-revision-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const OfflinePlugin = require('offline-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
+const { VueLoaderPlugin } = require('vue-loader');
+const WebpackPwaManifest = require('webpack-pwa-manifest');
+const gitRevisionPlugin = new GitRevisionPlugin();
 
 module.exports = {
+    mode: 'none',
```

Comment:
curious what mode: 'none' does here

Reply:
Ok, so, "mode" needs to exist to to tell webpack if we're prod or dev. However, we set those by variables in the scripts section, so it's (i think) redundant here. hence "none"

It changes how webpack applies certain optimisations.

Reply:
I know about mode, just haven't seen the property 'none'. if you skip it, default is mode: 'production', so you don't want to do that I think